streams.yml

```diff
@@ -195,3 +195,5 @@ centos_stream9:
   upstream_image: registry.ci.openshift.org/ocp/builder:stream9
   mirror: true
 
+bootc-rhel9:
+  image: registry.redhat.io/rhel9/rhel-bootc:9.4
```

Comment:
Noting that this is the first floating tag in streams.yml. On the one hand, it makes sense. On the other, this is a departure from how we use streams.yml. 

Maybe this warrants a comment that this should not be used as a pattern elsewhere?

Reply:
right, the validator is also failing, will fix that. Also check images:streams isn't misbehaving when it sees this. And leave a comment

Reply:
Added comment